cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,11 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-    
+    cy.visit('https://react-redux.realworld.io');
+    cy.get(':nth-child(2) > .nav-link').click();
+    cy.get(':nth-child(1) > .form-control').type('cypresstest1@test.com');
+    cy.get(':nth-child(2) > .form-control').type('Password_1');
+    cy.get('.btn').click();
```

Comment:
Better to specify this button by 'cy.contains' IMHO

Reply:
Suppose so, I took that into consideration with the second one, but still - ain't broke )